aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -1048,15 +1048,16 @@ Tensor zeros(IntArrayRef size,
     c10::optional<ScalarType> dtype,
     c10::optional<Layout> layout,
     c10::optional<Device> device,
-    c10::optional<bool> pin_memory) {
+    c10::optional<bool> pin_memory,
+    c10::optional<c10::MemoryFormat> optional_memory_format) {
   // See [Note: hacky wrapper removal for TensorOptions]
   TensorOptions options = TensorOptions().dtype(dtype).layout(layout).device(device).pinned_memory(pin_memory);
 
-  auto result = at::empty(size, options);
+  auto result = at::empty(size, options, optional_memory_format);
   return result.zero_();
 }
 
-Tensor& zeros_out(IntArrayRef size, Tensor& result) {
+Tensor& zeros_out(IntArrayRef size, c10::optional<c10::MemoryFormat> optional_memory_format, Tensor& result) {
```

Comment:
I don't think we want the out= variant of `zeros` to get a memory_format argument - you can also see that this variant doesn't take in most of the other parameters, like dtype/layout/device - they're all inferred from the passed in out tensor (`result`).